.devcontainer/devcontainer.json

```diff
@@ -20,9 +20,6 @@
 	],
 	"settings": {
 		"terminal.integrated.shell.linux": "/bin/bash",
-		"remote.extensionKind": {
-			"ms-azuretools.vscode-docker": "workspace"
-		},
```

Comment:
@philliphoff I removed the extension from container config. please review the others and let me know if I need to revise it.

Reply:
LGTM.